client/scripts/DrawTogether.js

```diff
@@ -596,8 +599,16 @@ DrawTogether.prototype.changeRoom = function changeRoom (room, number, x, y, spe
 			this.paint.addPublicDrawings(this.decodeDrawings(drawings));
 			this.chat.addMessage("Invite people", "http://www.anondraw.com/#" + room + number);
 			
-			if (this.account.uKey) {
+			if (typeof this.reputation === 'undefined') {//not logged in
+				setTimeout(function(){
```

Comment:
Copy pasted from the other one:

This is not a good way to do it. Randomly waiting gives very very very very very big problems down the line.
If you want to do something when the server knows our rep; just put it in the event handler where we get our reputation.